### PR TITLE
Copy_table outside

### PR DESCRIPTION
**Pull request by ue71603**
As the copy is once, I did take it out of the with statement. That seems to work

Now I get a different error in gtfs_db_to_db.py
```

ATTACH IF NOT EXISTS '<netexio.database.Database object at 0x00000190C0F020C0>' AS db_read (READ_ONLY);
Process ID: 16328
2024-12-16 14:14:58,023 - script_runner - ERROR - 'utf-8' codec can't decode byte 0xfc in position 99: invalid start byte: Traceback (most recent call last):
  File "C:\Users\ue71603\MG_Daten\github\reference1\gtfs-netex-test\tool_script_runner.py", line 216, in <module>
    main(args.script_file, args.log_file, args.log_level, args.blockname, args.begin_step)
  File "C:\Users\ue71603\MG_Daten\github\reference1\gtfs-netex-test\tool_script_runner.py", line 176, in main
    result=load_and_run(script_name, script_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ue71603\MG_Daten\github\reference1\gtfs-netex-test\tool_script_runner.py", line 41, in load_and_run
    result = mod(*args1)
             ^^^^^^^^^^^
  File "C:\Users\ue71603\MG_Daten\github\reference1\gtfs-netex-test\gtfs_db_to_db.py", line 32, in main
    copy_table(db_read, db_write,[DataSource, Codespace, StopPlace, PassengerStopAssignment, ScheduledStopPoint, AvailabilityCondition], clean=True)
  File "C:\Users\ue71603\MG_Daten\github\reference1\gtfs-netex-test\netexio\dbaccess.py", line 595, in copy_table
    db_write.con.execute(f"ATTACH IF NOT EXISTS '{db_read_file}' AS db_read (READ_ONLY);")
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xfc in position 99: invalid start byte

2024-12-16 14:14:58,023 - script_runner - ERROR - 'utf-8' codec can't decode byte 0xfc in position 99: invalid start byte: Traceback (most recent call last):
  File "C:\Users\ue71603\MG_Daten\github\reference1\gtfs-netex-test\tool_script_runner.py", line 216, in <module>
    main(args.script_file, args.log_file, args.log_level, args.blockname, args.begin_step)
  File "C:\Users\ue71603\MG_Daten\github\reference1\gtfs-netex-test\tool_script_runner.py", line 176, in main
    result=load_and_run(script_name, script_args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\ue71603\MG_Daten\github\reference1\gtfs-netex-test\tool_script_runner.py", line 41, in load_and_run
    result = mod(*args1)
             ^^^^^^^^^^^
  File "C:\Users\ue71603\MG_Daten\github\reference1\gtfs-netex-test\gtfs_db_to_db.py", line 32, in main
    copy_table(db_read, db_write,[DataSource, Codespace, StopPlace, PassengerStopAssignment, ScheduledStopPoint, AvailabilityCondition], clean=True)
  File "C:\Users\ue71603\MG_Daten\github\reference1\gtfs-netex-test\netexio\dbaccess.py", line 595, in copy_table
    db_write.con.execute(f"ATTACH IF NOT EXISTS '{db_read_file}' AS db_read (READ_ONLY);")
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xfc in position 99: invalid start byte


Process finished with exit code 0
```

I tried to see, where the UTF-8 might have gone wrong, but not found. Some encoding="utf-8" might be not needed. Perhaps you have an idea.

### CODE REVIEW

File: gtfs-netex-test/configuration.py [hunk 1]
```diff
@@ -20,7 +20,8 @@
 defaults["os"]= "windows"
 input_dir = "d:/input"
 processing_data =  "d:/aux_testing_processing"
-
+input_dir = "C:/Users/ue71603/MG_Daten/conversion/input"
```
Comment (skinkie) [inline gtfs-netex-test/configuration.py:23]: Nope, not absolute paths.

File: gtfs-netex-test/epip_db_to_db.py [hunk 2]
```diff
@@ -41,9 +41,9 @@ def main(source_database_file: str, target_database_file: str):
     with Database(target_database_file, read_only=False) as target_db:
         setup_database(target_db, classes, True)
         # attach_source(con, source_database_file) does not work persistently, requires an attach at every connection
+        copy_table(source_database_file, target_db, [Notice, ScheduledStopPoint], clean=True)
```
Comment (skinkie) [inline gtfs-netex-test/epip_db_to_db.py:44]: Don't want to go in this direction.

File: gtfs-netex-test/netexio/dbaccess.py [hunk 3]
```diff
@@ -585,12 +586,17 @@ def _prepare2(generator2, objectname):
 
     print('\n')
 
-def copy_table(db_read: Database, db_write: Database, classes: list, clean=False):
-    if db_read.read_only and os.name != 'nt':
-        print (f"ATTACH IF NOT EXISTS '{db_read.database_file}' AS db_read (READ_ONLY);")
-        db_write.con.execute(f"ATTACH DATABASE '{db_read.database_file}' AS db_read (READ_ONLY);")
+def copy_table(db_read_file: str, db_write: Database, classes: list, clean=False):
```
Comment (skinkie) [inline gtfs-netex-test/netexio/dbaccess.py:589]: Don't want to go in this direction.

File: gtfs-netex-test/configuration.py (hunk 1 above)
Comment (ue71603) [reply gtfs-netex-test/configuration.py:23]: I move this to local_config. those are my paths.

File: gtfs-netex-test/epip_db_to_db.py (hunk 2 above)
Comment (ue71603) [reply gtfs-netex-test/epip_db_to_db.py:44]: but it worked :-)

Comment (skinkie) [reply gtfs-netex-test/epip_db_to_db.py:44]: Yes, it does open the database, but it is read_only. But it does not make any sense that the _simple example_ (mail) does run, and the code below does not.

**Comment by skinkie** [comment]
Not required anymore.

**Pull request closed without merging**